specification/cognitiveservices/data-plane/Language/stable/2022-05-01/examples/questionanswering/authoring/SuccessfulProjectSubmitDeployJob.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Content-Type": "application/json",
+    "api-version": "2022-05-01",
+    "projectName": "proj1",
+    "deploymentName": "production"
+  },
+  "responses": {
+    "202": {
+      "description": "A successful call results with an Operation-Location header used to check the status of the analysis job.",
+      "headers": {
+        "Operation-Location": "job1"
```

Comment:
This seems to be wrong. Needs to be full URI of the job, including the right api-version.

Reply:
Changed